ops/c/include/ops_internal2.h

```diff
@@ -287,6 +287,20 @@ void ops_put_data(ops_dat dat);
 OPS_FTN_INTEROP
 void create_kerneldesc_and_enque(char const *name, ops_arg *args, int nargs, int index, int dim, int isdevice, int *range, ops_block block, void (*func)(struct ops_kernel_descriptor *desc));
 
+
+/*******************************************************************************
+* Random number generations
+*******************************************************************************/
+void ops_randomgen_init(unsigned int seed, int options);
```

Comment:
These ones (not the host ones) should go to ops_lib_core.h

Reply:
ok